Make bulk_extractor compile under Kali Linux

@zdavatz -THis is for you.
@simsong will use this distribution: https://aws.amazon.com/marketplace/pp/prodview-fznsw3f7mq7to

Ok, `configure` goes through, but `make -j9` still fails.
```
 495 |         if ( std::filesystem::exists( sc.outdir/"report.xml" )){
      |                   ^~~~~~~~~~
bulk_extractor.cpp:495:42: Fehler: »struct scanner_config« hat kein Element namens »outdir«
  495 |         if ( std::filesystem::exists( sc.outdir/"report.xml" )){
      |                                          ^~~~~~
bulk_extractor.cpp:504:37: Fehler: »struct scanner_config« hat kein Element namens »input_fname«
  504 |         p = image_process::open( sc.input_fname, cfg.opt_recurse, cfg.opt_pagesize, cfg.opt_marginsize );
      |                                     ^~~~~~~~~~~
bulk_extractor.cpp:504:104: Fehler: »image_process::open« kann nicht als Funktion verwendet werden
  504 |         p = image_process::open( sc.input_fname, cfg.opt_recurse, cfg.opt_pagesize, cfg.opt_marginsize );
      |                                                                                                        ^
bulk_extractor.cpp:507:73: Fehler: »struct scanner_config« hat kein Element namens »input_fname«
  507 |         cerr << "error: file " << e.what() << " is in directory " << sc.input_fname << std::endl;
      |                                                                         ^~~~~~~~~~~
bulk_extractor.cpp:511:71: Fehler: »struct scanner_config« hat kein Element namens »input_fname«
  511 |         cerr << "       then place them in a sub directory of " << sc.input_fname << std::endl;
      |                                                                       ^~~~~~~~~~~
bulk_extractor.cpp:539:50: Fehler: »struct scanner_config« hat kein Element namens »outdir«
  539 |     dfxml_writer *xreport = new dfxml_writer( sc.outdir / Phase1::REPORT_FILENAME, false ); // do not make DTD
      |                                                  ^~~~~~
bulk_extractor.cpp:547:23: Fehler: »struct scanner_config« hat kein Element namens »input_fname«
  547 |     validate_path( sc.input_fname );
      |                       ^~~~~~~~~~~
bulk_extractor.cpp:547:5: Fehler: »validate_path« wurde in diesem Gültigkeitsbereich nicht definiert
  547 |     validate_path( sc.input_fname );
      |     ^~~~~~~~~~~~~
bulk_extractor.cpp:565:38: Fehler: »struct scanner_config« hat kein Element namens »input_fname«
  565 |         cout << "Input file: " << sc.input_fname << std::endl ;
      |                                      ^~~~~~~~~~~
bulk_extractor.cpp:566:44: Fehler: »struct scanner_config« hat kein Element namens »outdir«
  566 |         cout << "Output directory: " << sc.outdir << std::endl ;
      |                                            ^~~~~~
bulk_extractor.cpp:600:46: Fehler: »struct scanner_config« hat kein Element namens »input_fname«
  600 |     xreport->xmlout( "provided_filename", sc.input_fname ); // save this information
      |                                              ^~~~~~~~~~~
make[2]: *** [Makefile:1438: scan_aes.o] Fehler 1
make[2]: *** [Makefile:1438: bulk_extractor_scanners.o] Fehler 1
make[2]: *** [Makefile:1438: notify_thread.o] Fehler 1
make[2]: *** [Makefile:1438: bulk_extractor.o] Fehler 1
make[2]: Verzeichnis „/home/zeno/.software/bulk_extractor/src“ wird verlassen
make[1]: *** [Makefile:526: all-recursive] Fehler 1
make[1]: Verzeichnis „/home/zeno/.software/bulk_extractor“ wird verlassen
make: *** [Makefile:466: all] Fehler 2
```

--- a/.github/workflows/ci-cd.yml
+++ b/.github/workflows/ci-cd.yml
@@ -10,6 +10,35 @@ permissions:
   contents: read
 
 jobs:
+  debian:
+    name: Debian Bookworm build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout recursively
+        uses: actions/checkout@v5
+        with:
+          submodules: recursive
+
+      - name: Bootstrap and build in Debian
+        run: |
+          docker run --rm \
+            --mount type=bind,source="$GITHUB_WORKSPACE",target=/src \
+            --workdir /src \
+            debian:bookworm \
+            bash -ec '
+              export DEBIAN_FRONTEND=noninteractive
+              apt-get update
+              apt-get install -y --no-install-recommends \
+                autoconf automake build-essential flex git libabsl-dev \
+                libexpat1-dev libgcrypt20-dev libgpg-error-dev libre2-dev \
+                libssl-dev libtool make pkg-config procps python3 zlib1g-dev
+              git config --global --add safe.directory /src
+              bash bootstrap.sh
+              ./configure --disable-libewf --enable-silent-rules --quiet
+              make -j2
+            '
+
   build:
     strategy:
       fail-fast: false

